orchagent/orchdaemon.cpp

```diff
@@ -77,8 +77,17 @@ bool OrchDaemon::init()
             APP_VNET_RT_TABLE_NAME,
             APP_VNET_RT_TUNNEL_TABLE_NAME
     };
-    VNetOrch *vnet_orch = new VNetOrch(m_applDb, APP_VNET_TABLE_NAME);
+    VNetOrch *vnet_orch;
+    if (platform == MLNX_PLATFORM_SUBSTRING)
+    {
+        vnet_orch = new VNetBitmapOrch(m_applDb, APP_VNET_TABLE_NAME);
```

Comment:
Please use EXEC Model instead of creating different orch. Lets have one VnetOrch